dags/graphics_telemetry.py

```diff
@@ -0,0 +1,115 @@
+import datetime
+
+from airflow import DAG
+from airflow.contrib.hooks.aws_hook import AwsHook
+from airflow.operators.sensors import ExternalTaskSensor
+from airflow.operators.subdag_operator import SubDagOperator
+
+from utils.dataproc import moz_dataproc_pyspark_runner, get_dataproc_parameters
+
+default_args = {
+    "owner": "bewu@mozilla.com",
+    "depends_on_past": False,
+    "start_date": datetime.datetime(2020, 11, 26),
+    "email": [
+        "telemetry-alerts@mozilla.com",
+        "bewu@mozilla.com",
+    ],
```

Comment:
Can we add here someone from platform team who's actually using this data?

Reply:
Yes forgot to do that